pymc3/backends/report.py

```diff
@@ -0,0 +1,145 @@
+from collections import namedtuple
+import logging
+
+
+logger = logging.getLogger('pymc3')
+
+
+SamplerWarning = namedtuple(
```

Comment:
might make sense to have `SamplerWarning.kind` be an `Enum`, so that there is a little more reuse?

Reply:
yes, fixed